src/content/docs/style-guide/writing-strategies/install-docs-guidance.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Choose from our install methods
+topics:
+  - New Relic only
+  - Basic style guide
+  - Writing guidelines
+---
+
+Installing New Relic is a crucial piece in our customer's experience. A failure to install something not only prevents our customers from using New Relic, but it almost certainly guarantees they won't be our customers eventually. Whether it's installing APM agents, the infrastructure agent, sending us logs, or installing CodeStream, among other things.
+
+This doc is intended to help you decide where to send your reader when they want to install something.
+
+## Choose the best install method [#choose]
+
+<table>
```

Comment:
Do you think this is more "naturally" readable with the header row on the bottom? It might mirror the reading experience of a flow graph but as a table.

Reply:
Interesting idea! I'll try it out and see what it looks like